### PR TITLE
Feature/rqt gait selection

### PR DESCRIPTION
**Pull request by Ishadijcks**
Add an intuitive UI based on #203, which should be merged before this one.

How to run:

Start the Gait Selection:
```
roslaunch march_gait_selection march_gait_selection.launch 
```
Start the rqt plugin:
```
roslaunch march_rqt_gait_selection march_rqt_gait_selection.launch 
```

Please confirm that the following works:
- Basic stuff
  - The ui builds correctly.
- Applying versions
  - Change a subgait and press the apply button.
  - Restart the rqt plugin
  - Confirm the previously selected subgait is still there.
  - Kill the rqt plugin
  - Restart the gait selection
  - Start the rqt plugin
  - Confirm the default selected subgait is back.
- Saving defaults
  - Change a subgait and press the save defaults button.
  - Restart the rqt plugin
  - Confirm the previously selected subgait is still there.
  - Kill the rqt plugin
  - Restart the gait selection
  - Start the rqt plugin
  - Confirm the previously selected subgait is still there.

### CODE REVIEW

File: march_rqt_gait_selection/launch/march_rqt_gait_selection.launch [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<launch>
+    <node name="march_rqt_gait_selection" pkg="march_rqt_gait_selection" type="rqt_gait_selection" output="screen"/>
+</launch>
```
Comment (Ishadijcks) [inline march_rqt_gait_selection/launch/march_rqt_gait_selection.launch]:
```suggestion
</launch>

```

File: march_rqt_gait_selection/launch/march_rqt_gait_selection.launch [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<launch>
```
Comment (TimBuckers) [inline march_rqt_gait_selection/launch/march_rqt_gait_selection.launch]: Not sure what our naming convention is, but we could name the launch file: rqt_gait_selection.launch

File: march_rqt_gait_selection/resource/gait_selection.ui [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      <enum>QFrame::StyledPanel</enum>
+     </property>
+     <property name="frameShadow">
+      <enum>QFrame::Raised</enum>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout">
+      <item>
+       <widget class="QPushButton" name="Refresh">
+        <property name="text">
+         <string>Refresh</string>
```
Comment (JitskedeVries) [inline march_rqt_gait_selection/resource/gait_selection.ui]: Maybe 'reset to default' is more descriptive of what the button does

File: march_rqt_gait_selection/launch/march_rqt_gait_selection.launch (hunk 2 above)
Comment (Ishadijcks) [reply march_rqt_gait_selection/launch/march_rqt_gait_selection.launch]: This is our naming convention

**Pull request merged**